mathics/core/list.py

```diff
@@ -83,7 +83,7 @@ def __repr__(self) -> str:
 
     def __str__(self) -> str:
         """str() representation of ListExpression. May be longer than repr()"""
-        return f"<ListExpression: {self._elements}>"
+        return "{" + ",".join(e.__str__() for e in self._elements) + "}"
```

Comment:
See comments above with respect to using underscore functions. (Yes, what was here before was also probably not conventionally used.)